Jenkinsfile

```diff
@@ -19,11 +19,11 @@ elifePipeline {
                     'behat': '/srv/journal/build/behat/*.xml'
                 ])
 
-                try {
-                    sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml up -d cli fpm web"
-                } finally {
-                    sh 'docker-compose -f docker-compose.yml -f docker-compose.ci.yml down --volumes'
-                }
+                dockerComposeSmokeTests('journal', commit, [
+                    'scripts': [
+                        'cli': './smoke_tests_cli.sh',
```

Comment:
adding `down --volumes` here too, it's the last `dockerCompose*Tests` custom step

Reply:
Already opened https://github.com/elifesciences/elife-jenkins-workflow-libs/pull/5.